src/napari_workflows/_workflow.py

```diff
@@ -17,22 +25,31 @@ class Workflow():
     the result of the specified task.
     """
 
+    _tasks: dict[str, Any]
+    """
+    A task is either a callable task with type: `tuple[Callable, Any, ...]` where the latter arguments are arguments to the function
+    Alternatively, a task can be a data task, which can have any other type.
+    Unfortunately these types can't really be expressed in the current type system
+    """
+
```

Comment:
This isn't a class variable, it's a type annotation for an instance variable. It's part of PEP 526: https://peps.python.org/pep-0526/#class-and-instance-variable-annotations.

Your example is different, because you actually assign a value to the variable using `=` which makes it a class variable. I'm not doing that.